docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.8"
+
+services:
+  mongodb:
+    image: mongo:5.0.2
+    restart: unless-stopped
+    env_file: ./.env
```

Comment:
Do we need this? I don't think it's necessary, and I think it would give an error if the file doesn't exist, right?